include/passwd_notify.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+namespace passwdnotify {
+
+class PasswdNotify {
```

Comment:
Why would you even put in a class if there's no state...

Reply:
It's my personal preference in C++/Java (to easily know which functions are public functions).

I'll remove the class statement.

Reply:
If it's not static, it's globally visible...